Yoast/javascript
# FRO-11 update styling form fields settings pages

## Summary


This PR can be summarized in the following changelog entry:

* [@yoast/components] Update the input fields, text areas, dropdowns,  buttons, title separator and toggle switches in the settings pages to the new styling

## Relevant technical choices:

*
## Test instructions

This PR can be tested by following these steps:

* Pull the 14291-update-styling-form-fields in from the wordpress-seo repo
* Pull this repo
* Link the javascript repo
* Build the wordpress-seo
* Check the input fields, text areas, dropdowns,  buttons, title separator and toggle switches are in line with the design: https://sketch.cloud/s/KLz2A/a/W0Zj00/play

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/14291
